src/Map.js

```diff
@@ -20,6 +20,20 @@ const Map = function() {
 
   let mapContainer;
 
+  let loadMarkers = async (map) => {
+    return new Promise((resolve, reject) => {
+      map.loadImage('marker.png', (error, image) => {
+        if (error) {
+          reject(error);
+          return;
+        }
+
+        map.addImage('marker', image);
+        resolve();
+      });
+    });
+  }
```

Comment:
Instead of hardcoding `markers.png` We could pass a list of images to load to this function:

```
let loadMarkers = async (map, icons) => {
  ...
```

The list of icons to load can be taken  from `points.json`